### ISSUE TITLE
Matroska cover art 

### ISSUE THREAD

**Issue by zyw80** [Martchus/tageditor#55]
You can't edit cover art in matroska files: it shows only as attachment, but you can't view/add/remove or renaming anything. Are you planning to add this function?

**Comment by Martchus** [comment]
Right, so far one can add/remove/extract the cover only as attachment and one has to be aware of [the guidelines](https://matroska.org/technical/attachments.html). It would of course be possible to *somehow* map the cover attachment into a tag's cover to the way the cover is handled wouldn't differ to other formats from the user perspective.

Not sure whether it is worth implementing this feature because it might be quite complicated and raises certain questions, e.g.: What happens when a Matroska file has an attachment but no tags? And if a Matroska file has multiple tags with which tag should the cover be associated? How to make it clear that the cover is actually *not* part of a tag and therefore the target level of the tag does not apply to it?
